src/routes/(infoPages)/about/events/shared.server.ts

```diff
@@ -0,0 +1,110 @@
+import { CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+import getContentfulClient from "$lib/services/contentful";
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+import type { EventsQuery } from "./$queries.generated";
+import { error } from "@sveltejs/kit";
+import type { PageServerLoad } from "./page/[page]/$types";
+import type { Breadcrumbs } from "$lib/components/Breadcrumbs";
+import { events as testEvents, pages as testEventPages } from "./__tests__/eventsTestContent";
+
+const limit = 20;
+
+export const query = gql`
+  query Events($limit: Int = 20, $skip: Int = 0) {
+    eventEntryCollection(limit: $limit, skip: $skip, order: [eventDateAndTime_DESC]) {
```

Comment:
Not sure if this is the appropriate place, but we'll want to improve this query so that we're filtering out past events and sorting in ascending order.